src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -47,30 +47,27 @@ class LocalBackend extends Backend with LazyLogging {
   override def executeCommand(commandLine: String, workflowDescriptor: WorkflowDescriptor, call: Call, scopedLookupFunction: ScopedLookupFunction): Try[Map[String, WdlValue]] = {
     import LocalBackend._
 
-    val callDirectory = Paths.get(hostExecutionPath(workflowDescriptor).toFile.getAbsolutePath, "call-" + call.name).toFile
-    // FIXME mkdir*s* should not be required here, the directory structure up to the call should
-    // FIXME already exist and must recently have been perturbed.
-    callDirectory.mkdirs()
+    val hostCallDirectory = Paths.get(hostExecutionPath(workflowDescriptor).toFile.getAbsolutePath, "call-" + call.name).toFile
+    hostCallDirectory.mkdirs()
 
-    val (stdoutFile, stdoutWriter) = FileUtil.tempFileAndWriter("stdout", callDirectory)
-    val (stderrFile, stderrWriter) = FileUtil.tempFileAndWriter("stderr", callDirectory)
-    val (commandFile, commandWriter) = FileUtil.tempFileAndWriter("command", callDirectory)
+    val (stdoutFile, stdoutWriter) = FileUtil.tempFileAndWriter("stdout", hostCallDirectory)
+    val (stderrFile, stderrWriter) = FileUtil.tempFileAndWriter("stderr", hostCallDirectory)
+    val (commandFile, commandWriter) = FileUtil.tempFileAndWriter("command", hostCallDirectory)
 
     val hostWorkflowExecutionFile = hostExecutionPath(workflowDescriptor).toFile
     val containerExecutionDir = s"/root/${workflowDescriptor.id.toString}"
 
-    if (call.docker.isEmpty) {
-      val callDirectory = Paths.get(hostWorkflowExecutionFile.getAbsolutePath, s"call-${call.name}")
-      commandWriter.writeWithNewline(s"cd $callDirectory")
-    }
+    val parentPath = if (call.docker.isDefined) Paths.get(containerExecutionDir).toString else hostWorkflowExecutionFile.getAbsolutePath
+    val callDirectory = Paths.get(parentPath, s"call-${call.name}")
+    commandWriter.writeWithNewline(s"cd $callDirectory")
+
     commandWriter.writeWithNewline(commandLine)
     commandWriter.flushAndClose()
 
     def buildDockerRunCommand(image: String): String =
       // -v maps the host workflow executions directory to /root/<workflow id> on the container.
-      // -w sets the command's working directory on the container to /root/<workflow id>/call-<call name>.
       // -i makes the run interactive, required for the cat and <&0 shenanigans that follow.
-      s"docker run -v ${hostWorkflowExecutionFile.getAbsolutePath}:$containerExecutionDir -w $containerExecutionDir/call-${call.name} -i $image"
+      s"docker run -v ${hostWorkflowExecutionFile.getAbsolutePath}:$containerExecutionDir -i $image"
```

Comment:
In the previous code, the non-Docker path would execute as the first line of the command script:

```
cd cromwell-executions/workflow-name/UUID/call-dir
```
while the Docker path would do the `-w /root/UUID/call-dir` to get similar behavior.  This change harmonizes the two paths to both do the cd in the command script.